<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkSegmentationPreferencePageControls</class>
 <widget class="QWidget" name="QmitkSegmentationPreferencePageControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>779</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="slimViewLabel">
     <property name="text">
      <string>Slim view</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="slimViewCheckBox">
     <property name="text">
      <string>Hide tool button texts and increase icon size</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="displayLabel">
     <property name="text">
      <string>2D display</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="displayOptionsLayout">
     <item>
      <widget class="QRadioButton" name="outlineRadioButton">
       <property name="text">
        <string>Draw as outline</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">displayButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="overlayRadioButton">
       <property name="text">
        <string>Draw as transparent overlay</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">displayButtonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="selectionModeLabel">
     <property name="text">
      <string>Data node selection mode</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="selectionModeCheckBox">
     <property name="toolTip">
      <string>If checked the segmentation plugin ensures that only the selected segmentation and the reference image are visible at one time.</string>
     </property>
     <property name="text">
      <string>Show only selected nodes</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="surfaceCreationLabel">
     <property name="text">
      <string>Smoothed surface creation</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QFormLayout" name="surfaceLayout">
     <property name="horizontalSpacing">
      <number>8</number>
     </property>
     <property name="verticalSpacing">
      <number>8</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="smoothingLabel">
       <property name="text">
        <string>Smoothing value (mm)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="smoothingSpinBox">
       <property name="toolTip">
        <string>The Smoothing value is used as variance for a gaussian blur.</string>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="decimationLabel">
       <property name="text">
        <string>Decimation rate</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="decimationSpinBox">
       <property name="toolTip">
        <string>Valid range is [0, 1). High values increase decimation, especially when very close to 1. A value of 0 disables decimation.</string>
       </property>
       <property name="maximum">
        <double>0.990000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="closingLabel">
       <property name="text">
        <string>Closing Ratio</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="closingSpinBox">
       <property name="toolTip">
        <string>Valid range is [0, 1]. Higher values increase closing. A value of 0 disables closing.</string>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QCheckBox" name="smoothingCheckBox">
       <property name="toolTip">
        <string>If checked the segmentation plugin ensures that only the selected segmentation and the reference image are visible at one time.</string>
       </property>
       <property name="text">
        <string>Use image spacing as smoothing value hint</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelSetPresetLabel">
     <property name="text">
      <string>Default label set preset</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLineEdit" name="labelSetPresetLineEdit">
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="labelSetPresetToolButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="labelSetPresetCmdLineArgLabel">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;The default label set preset is currently overriden by the &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; color:#ff0000;&quot;&gt;Segmentation.labelSetPreset&lt;/span&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt; command-line argument.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelCreationLabel">
     <property name="text">
      <string>Label creation</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="defaultNameRadioButton">
       <property name="text">
        <string>Assign default name and color</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">labelCreationButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="askForNameRadioButton">
       <property name="text">
        <string>Ask for name and color</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">labelCreationButtonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="labelSuggestions">
     <property name="text">
      <string>Label suggestions</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLineEdit" name="suggestionsLineEdit">
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="suggestionsToolButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="suggestionsCmdLineArgLabel">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;Suggestions are currently enforced by the &lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; color:#ff0000;&quot;&gt;Segmentation.labelSuggestions&lt;/span&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt; command-line argument.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="1">
    <widget class="QCheckBox" name="replaceStandardSuggestionsCheckBox">
     <property name="text">
      <string>Replace standard organ suggestions</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QCheckBox" name="suggestOnceCheckBox">
     <property name="text">
      <string>Suggest once per segmentation</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="displayButtonGroup"/>
  <buttongroup name="labelCreationButtonGroup"/>
 </buttongroups>
</ui>
